src/extendStyles.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint react/forbid-foreign-prop-types: off */
+
+import deepmerge from 'deepmerge';
+
+function validateStyle(style, extendableStyles, currentPath = '') {
+  if (process.env.NODE_ENV !== 'production') {
+    if (style === null || Array.isArray(style) || typeof style !== 'object') {
+      return;
```

Comment:
why is this returning early if it's an array, or a non-object? should this throw instead?

Reply:
If we get a `string` or `number`, we have reached a style's value and don't need to validate any further.
```
{
  container: {
    color: 'red', // 'red' is value, stop recursively validating
  }
}
```

I'll update to check for string || number.